src/SUMMARY.md

```diff
@@ -39,6 +39,7 @@
 
 - [Rust 2024 🚧](rust-2024/index.md)
   - [Additions to the prelude](rust-2024/prelude.md)
+  - [Unsafe functions](rust-2024/unsafe-functions.md)
```

Comment:
Perhaps we might want to say "Newly unsafe functions" to correspond to e.g. "*Additions* to the prelude"?